#include <Python.h>

extern PyObject* lightship_error;
